src/download/exchangeDownloader.ts

```diff
@@ -228,20 +230,15 @@ export async function searchExchange(
  * @export
  * @param {string} accessToken
  * @param {RestApi} restApi
- * @param {RegExp} [deployment]
  * @returns {Promise<string>} Returned the version string from the instance fetched asset.version value
  */
-export async function getVersionByDeployment(
+export async function getApiVersions(
```

Comment:
It's not clear in the description or the function name that we get the latest version, we should rename this function to something like `getLatestApiVersions` and/or update the JS doc description

Reply:
Refactored to put all the new code behind isOas flag